Manual File Locking - Lock Breakers (#4513)
owncloud/docs: modules/user_manual/pages/files/manual_file_locking.adoc

== Using Manual File Locking

> NOTE: The _user-facing_ components in the web interface are disabled by default and must be enabled by the ownCloud admininstrator, because this feature allows users to lock other users files *exclusively*. Lock releasing can only be done by the locking user, a user being member of a lock breaking group if defined by the ownCloud administrator or until the lock expires.

Reviewer: ```suggestion
NOTE: The _user-facing_ components in the web interface are disabled by default and must be enabled by the ownCloud administrator, because this feature allows users to lock other users files *exclusively*. Lock releasing can only be done by the locking user, a user who is a member of a lock breaking group (if defined by the ownCloud administrator) or until the lock expires.
```
